app/assets/stylesheets/jobs.css

```diff
@@ -0,0 +1,24 @@
+.jobs {
+  float: right;
+  width: 80%;
+}
+
+table {
+  width: 100%;
+}
+
+.index-job {
+  width: 100%;
+  border: solid;
+  border-color: black;
+  border-width: 1px;
+}
+
+.index-job:hover {
+  background-color: #BDC3C7;
+}
+
+.index-job-detail {
+  padding-left: 10px;
+  padding-right: 10px;
```

Comment:
Yeah, I had trouble remembering the order until I saw something that said to just remember that it's big TRouBLe (top-right-bottom-left). Now I never forget :smile: